src/main/scala/com/webtrends/harness/component/spray/authentication/OAuthDirectives.scala

```diff
@@ -28,7 +28,8 @@ class OAuthHttpAuthenticator[U](val scope: String, val oauthAuthenticator: OAuth
   }
 
   def getChallengeHeaders(httpRequest: HttpRequest) =
-    `WWW-Authenticate`(HttpChallenge(scheme = this.scheme, realm = scope, params = Map.empty)) :: Nil
+    // Returning Basic scheme here to make sure that we return a basic header to get the username/password dialog
+    `WWW-Authenticate`(HttpChallenge(scheme = "Basic", realm = scope, params = Map.empty)) :: Nil
```

Comment:
Is the hardcoding of this to "Basic" permanent, or temporary?  Does this logic only apply to basic auth?